### DISCUSSION TITLE
Enable OKTA in k8s dashboard

### DISCUSSION THREAD

**Discussion by josentt1** [kubernetes/dashboard#7989] [Q&A]
is this possible? any procedure or docummentation? saml or OpenIDConnect protocol? 

thanks!!

**Comment by floreks** [comment]
We do not provide any official documentation for OIDC/OAuth integrations. There are lots of different software that can be used for that. In general, the principle is always the same. You need to configure a reverse proxy in front of the Dashboard that will take care of auth process and only inject a valid token into the authorization header. More info can be found here: https://github.com/kubernetes/dashboard/tree/master/docs/user/access-control#authorization-header

What software you would like to use to achieve that is up to you.
